de/manual/ch_Transactions.docbook

```diff
@@ -2624,57 +2606,49 @@ Translators:
       </caution>
     </sect2>
 
-    <sect2 id="trans-import-MT940-MT942-DTAUS">
-      <title>Importieren von MT940 &amp; MT942 SWIFT sowie DTAUS Dateien</title>
+    <sect2 id="trans-import-aqbanking">
+      <title>Import verschiedener Dateiformate</title>
 
-      <para>In dem Menü &mc-gsm.fi.imp; stehen die folgenden Einträge zur Verfügung:
+      <para>Wenn Sie &app-aqb; auf Ihrem Computer installiert haben und &app; mit &app-aqb; Unterstützung
+        gebaut wurde, können Sie den Menüeitnrag &mc.fi.imp.AqB; verwenden, um Dateien in
+        verschiedenen Formaten zu importieren.
       </para>
-<!--FIXME replace with link to glossary if and when extended to help -->
-      <variablelist>
-        <varlistentry>
-          <term>&gmi.fi.imp.mt940;</term>
 
-          <listitem>
-            <para>Das Format SWIFT MT940 wird für einen Tagesabschluss von Banken verwendet, die das SWIFT-Netzwerk
-              für den Datenaustausch nutzen.
-            </para>
-          </listitem>
-        </varlistentry>
+      <para>Zunächst wird ein Dialogfeld mit zwei Listen angezeigt. Die linke Liste enthält Dateiformate und
+        die rechte Liste enthält die jeweils verfügbaren Importprofile mit spezifischen Vorlagen.
+      </para>
 
-        <varlistentry>
-          <term>&gmi.fi.imp.mt942;</term>
+      <figure pgwide="1">
+        <title>Dateiformate zum Import mit &app-aqb;</title>
 
-          <listitem>
-            <para>MT942 wird für einen Zwischenauszug im selben System verwendet. (in der Zeit seit dem letzten
-              Auszug gesendet)
-            </para>
-          </listitem>
-        </varlistentry>
+        <screenshot>
+          <mediaobject>
+            <imageobject role="html">
+              <imagedata fileref="figures/Import_AqB_Profile.png"  srccredit="Christian Wehling" width="&img-w;" />
+            </imageobject>
 
-        <varlistentry>
-          <term>&gmi.fi.imp.dtaus;</term>
+            <imageobject role="fo">
+              <imagedata fileref="figures/Import_AqB_Profile.png"  srccredit="Christian Wehling" />
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
 
-          <listitem>
-            <para>DTAUS ist ein Format das seit 1976 mit 9-Spur-Magnetbändern und später mit anderen Wechselmedien
-              (z.B. Floppy Disketten) verwendet wurde.
-            </para>
+      <para>Wählen Sie einen Dateiformat und ein Importprofil aus den Listen. &app; merkt sich Ihre Auswahl
```

Comment:
Ich denke, wir sollten, soweit bekannt, die Formate oben in Importformate erklären und hier einen Link einfügen.

Was mir beim Betrachten des Dialogs auffiel:
CaMt: Cash Management, Teil vom neueren, XML-basierten ISO-20022-Standard, Komplement:
SEPA:  PaIn: Payment Initiation (eigentlich nur Export)
SWIFT: Traditioneller zeilenorientierter, internationaler Standard mit MessageTypes MT9XXX zur Kundenkommunikation
https://de.wikipedia.org/wiki/SWIFT
XML: Doppelnennungen auf AB-Liste klären
yellownet: CH: PostFinance (traditionell). Irgendwann bekam Martin keine Infos mehr von denen. Später schickte uns ein Benutzer die Specs im Wiki, die nach ISO-20022 aussehen.
QIF: traditioneller Industriestandard nach Intuit's Quicken
OFX: offener Standard in Amerika gebräuchlich

CSV, OFX: Unterschied zwischen GnuCash- und AqB-Import?

Reply:
… und Links zu den WP-Artikeln der Formate sollten wir da einfügen.